packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -61,7 +63,7 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             collapsedNodes: [],
             orderedNodes: [],
             xyData: {},
-            columns: [{title: 'Name', sortable: true}]
+            columns: [{ title: 'Name', sortable: true }]
```

Comment:
There seem to be a lot of lines in the diff that are whitespace changes unrelated to adding navigation to XY charts. Unrelated code cleanup changes should at least be in a separate commit (and probably in a different PR). Including unrelated changes adds a lot of noise to the diff and makes the code more difficult to review.

When creating commits with your changes, I recommend reading over the diff and manually staging functional changes using a GUI git tool:
* If the internal git tool in your IDE doesn't allow staging diff sections or individual lines, you can use gitk to create commits.
* First, create a commit with the functional changes. 
* Then, any whitespace or style changes leftover can be committed separately. One way to create a separate PR for leftover style changes would be to "git stash" them, create a new branch from master, then reapply the changes (I think it's "git stash pop").